modules/ROOT/pages/create-partner.adoc

```diff
@@ -0,0 +1,71 @@
+= Configure a New Trading Partner
+
+Create a partner by specifying the partner's name, identifiers, endpoints, validation rules, and other settings.
+
+== Prerequisites
+
+Before you create your first partner, you must xref:setup-payload-storage-API.adoc[configure your payload storage API].
```

Comment:
Working to avoid modal verbs....
```suggestion
Before you create your first partner, xref:setup-payload-storage-API.adoc[configure your payload storage API].
```